Moderation-Policy.md

```diff
@@ -0,0 +1,80 @@
+
+### Policy
+
+* All Posts are expected to respect the Node.js Code of Conduct.
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Only a TSC member may Ban an individual from the Node.js Github Organization.
+* Any individual Banned from the Node.js Github Organization will be recommended for exclusion from any Node.js Foundation sponsored event or activity.
+
+#### Collaborator Posts
+
+* Collaborator's must not Moderate any Post authored by another Collaborator without first giving the author an opportunity of at least 24 hours (from the time of the initial request) to modify or remove the Post on their own.
+* In extreme circumstances the TSC can be consulted to waive the 24 hour grace period.
+* If the author of the Post disagrees that Moderation is required, the matter can be escalated to the TSC by creating a new issue (or using an existing one) within the nodejs/moderation repository with the label `tsc-agenda`. In such cases, the TSC will serve as the final arbiter.
+* For Moderation issues involving a TSC member, the member in question is expected to recuse themselves from any decisions required to resolve the issue.
+* When Moderating any Post authored by another Collaborator, the moderating Collaborator must:
+ * Explain the justification for Moderating the post,
+ * Identify all changes made to the Post, and
+ * Identify the steps previously taken to resolve the issue.
+* Explanations of Moderation actions on Collaborator Posts must be provided in:
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Any Collaborator who habitually authors Posts that must be Moderated will be subject to possible removal from the Node.js Github Organization. Such action can only be taken through normal TSC motion and vote.
+
+#### Non-Collaborator Posts
+
+* Posts authored by non-Collaborator's are always subject to immediate Moderation by any Collaborator if the content is intentionally disruptive or in violation of the Node.js Code of Conduct.
+* When Moderating non-Collaborator Posts, the moderating Collaborator should:
+ * Explain the justification for Moderating the post, and
+ * Identify all changes made to the Post.
+* Explanations of Moderation actions for non-Collaborator Posts must be provided in:
+ * The original Post being modified (as replacement or appended content),
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Moderation of Posts authored by non-Collaborators may result in those non-Collaborators being Banned from further participation in the Node.js Github Organization for an indefinite period of time.
+
+#### Requesting Moderation
+
+Requests to Moderate a Post are only necessary when the Post in question was written by a Collaborator.
```

Comment:
This may make sense to live higher in the document and specify various cases where requesting moderation is necessary. This could be a great place to outline our moderation / reporting policy for non-collaborators